test/hotspot/jtreg/gc/shenandoah/mxbeans/TestChurnNotifications.java

```diff
@@ -141,8 +165,8 @@ public void handleNotification(Notification n, Object o) {
                     Map<String, MemoryUsage> mapBefore = info.getGcInfo().getMemoryUsageBeforeGc();
                     Map<String, MemoryUsage> mapAfter = info.getGcInfo().getMemoryUsageAfterGc();
 
-                    MemoryUsage before = mapBefore.get("Shenandoah");
-                    MemoryUsage after = mapAfter.get("Shenandoah");
+                    MemoryUsage before = getUsage(mapBefore);
+                    MemoryUsage after = getUsage(mapAfter);
```

Comment:
This also changes test logic for traditional Shenandoah, but its harmless. More correct would be to require "Young Gen" pool to only exist for -XX:ShenandoahGCMode=generational.